src/js/components/ServicesBreadcrumb.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+import ServicesPathUtil from '../utils/ServicesPathUtil';
+
+const breadcrumbIcon = (
+  <i className="icon icon-sprite icon-sprite-small icon-back forward" />
+);
+
+var ServicesBreadcrumb = React.createClass({
+  displayName: 'ServicesBreadcrumb',
+
+  propTypes: {
+    appId: React.PropTypes.string,
+    groupId: React.PropTypes.string,
+    taskId: React.PropTypes.string,
+    volumeId: React.PropTypes.string
+  },
+
+  getRootLink: function () {
+    const {groupId, appId} = this.props;
+
+    if (groupId == null ||
+      (groupId === '/' && appId == null)) {
+      return (
+        <span className="crumb">Services</span>
+      );
+    }
+
+    return (
+      <span className="crumb">
+        <Link to="services">Services</Link>
+      </span>
+    );
+  },
+
+  getGroupLinks: function () {
+    var {appId, groupId} = this.props;
+
+    if (groupId == null) {
+      return null;
+    }
+
+    const pathParts = groupId.split('/').slice(0, -1);
```

Comment:
Please consider using a combination of ServiceTree `filterItems` and `reduceItems` in conjunction with Service `getId` to create a list of links instead of this string fiddling.